### PR TITLE
clean up nginx+ sensitive data after installation

### PR DESCRIPTION
**Pull request by azych**
* make sure we clean up nginx-plus sensitive data after installation in each Dockerfile: cert, keys, repo

### CODE REVIEW

**Review by apasyniuk — APPROVED** [review]

File: centos/Dockerfile [hunk 1]
```diff
@@ -48,11 +48,15 @@ RUN set -ex \
...
+  && sh ./install.sh -y \
+  # cleanup sensitive nginx-plus data
+  && rm /etc/ssl/nginx/nginx-repo.* \
+  && rm /etc/yum.repos.d/nginx-plus-7.4.repo \
+  && gpg --batch --delete-keys "$NGINX_GPGKEY"
 
 # Forward request logs to Docker log collector
 RUN ln -sf /dev/stdout /var/log/nginx-controller/agent.log \
-  && ln -sf /dev/stderr /var/log/nginx/error.log 
+  && ln -sf /dev/stderr /var/log/nginx/error.log
```
Comment (1996sajal) [inline centos/Dockerfile:59]: I think `SPACE` added by mistake.

File: debian/Dockerfile [hunk 2]
```diff
@@ -53,13 +53,16 @@ RUN set -ex \
...
+  && rm /etc/apt/sources.list.d/nginx-plus.list \
+  && rm /etc/apt/apt.conf.d/90nginx \
+  && apt-key del "$NGINX_GPGKEY"
 
 # Forward request logs to Docker log collector
 RUN ln -sf /dev/stdout /var/log/nginx-controller/agent.log \
-  && ln -sf /dev/stderr /var/log/nginx/error.log 
-
-
+  && ln -sf /dev/stderr /var/log/nginx/error.log
```
Comment (1996sajal) [inline debian/Dockerfile:65]: SAME.

**Review by 1996sajal — APPROVED** [review]
Looks good to me. 👍🏻

File: centos/Dockerfile (hunk 1 above)
Comment (azych) [reply centos/Dockerfile:59]: this change is **removing** the unnecessary space, not adding it

File: debian/Dockerfile (hunk 2 above)
Comment (azych) [reply debian/Dockerfile:65]: same here - my change is removing the excessive whitespace :)

**Pull request merged**